.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+      - alpha
```

Comment:
Not something for this PR but I'm thinking that maybe we should add `beta` and `next` here in case we decide to name the prerelease branch something else, like they did in [eik plugin for semantic-release](https://github.com/eik-lib/semantic-release/blob/master/.github/workflows/publish.yml#L7-L9)
Then we'd also need to tell semantic-release that those branches can be treated as a prerelease in `.releaserc.json`.
What do you think?